### PR TITLE
Add Getters to Bestuursorgaan and Mandaat to Check If Decretaal

### PR DESCRIPTION
**Pull request by ruben-bottu**

### CODE REVIEW

File: config/migrations/2024/20240216000000-decretaal-bestuursorgaan-codes.ttl
Comment (bfidlers) [inline config/migrations/2024/20240216000000-decretaal-bestuursorgaan-codes.ttl:1]: I'm a bit confused, these bestuursorganen seem to be different than the ones described in this PR: https://github.com/lblod/frontend-lokaal-mandatenbeheer/pull/61/files

**Review by bfidlers — COMMENTED** [review]
Maybe it would also be a good idea to document somewhere which ones are decretaal, because it is a kind of a headache to search this up

File: config/migrations/2024/20240216000000-decretaal-bestuursorgaan-codes.ttl
Comment (Rahien) [reply config/migrations/2024/20240216000000-decretaal-bestuursorgaan-codes.ttl:1]: indeed this confuses me too. how come they are not the same?

**Review by Rahien — CHANGES_REQUESTED** [review]
uuid and there is a discrepancy in uris used here and in the frontend pr

**Pull request merged**